strutil/pathiter.go

```diff
@@ -66,60 +66,73 @@ func (iter *PathIterator) Path() string {
 	return iter.path
 }
 
-// CurrentBase returns the name of the current path element.
-// It will only end with a slash if the iterator is currently
-// placed at the base of the filesystem.
-// Ex:
+// CurrentBase returns the last element of the current path.
+// It removes any trailing slash unless the last element of the
+// current path is the filesystem root (and so CurrentBase() == "/").
+// If the current path is empty (i.e. Next() has not yet been called),
```

Comment:
Nitpick: try to be more concise, perhaps something like: `Before any iteration with Next, CurrentBase returns an empty string.`